README.md

```diff
@@ -74,32 +76,32 @@ The [Raiden dApp](#raiden-dapp) is a reference implementation of the Raiden Ligh
 
 The [Raiden Network](https://raiden.network/) is an off-chain scaling solution, enabling near-instant, low-fee and scalable payments. It’s complementary to the Ethereum blockchain and works with any ERC20 compatible token.
 
-The Raiden client code is available [here](https://github.com/raiden-network/raiden) and has been [released for mainnet](https://medium.com/raiden-network/red-eyes-mainnet-release-announcement-d48235bbef3c) with a limited alpha release of the Raiden Network in December 2018.
+The Raiden client code is available [here](https://github.com/raiden-network/raiden) and has been [released for mainnet](https://medium.com/raiden-network/alderaan-mainnet-release-announcement-7f701e58c236).
 
 <center>
 <img 
       width='750px' 
       alt='' 
-      src="https://user-images.githubusercontent.com/43838780/67964837-438d2080-fc01-11e9-93a7-c38129b7d566.png" />
+      src="https://user-images.githubusercontent.com/43838780/85526750-8ecdc680-b60a-11ea-8a42-07da6b0f8296.png" />
 </center>
 
 The goal of the Raiden Light Client SDK is to provide an easy-to-use framework, which can be integrated by any JavaScript developer. The SDK will simplify the process of embedding and using the Raiden Network for token transfers in decentralized applications
 
-With the SDK we want to make your life as a dApp dev easier:
+With the SDK we want to make your life as a dApp developer easier by:
 
-- You should be able to interact with the Raiden Network easily with your dApp.
-- We want to help you to enable your users to make token transfers using their consumer wallets like imToken or Metamask.
-- It should be possible to send tokens using low end devices, which would not be capable of running a full Raiden node.
+- Providing an easy way for your dApp to interact with the Raiden Network.
```

Comment:
I would like to keep it in an active voice

see: https://developers.google.com/style/voice